tests/rundb/test_httpdb.py

```diff
@@ -494,6 +495,109 @@ def _create_feature_vector(name):
     }
 
 
+def test_tagging_artifacts(create_server):
+    _, db = _configure_run_db_server(create_server)
+
+    project = "newproj"
+    proj_obj = mlrun.new_project(project, save=False)
+    db.create_project(proj_obj)
+
+    tag = "my-tag"
+    add_tag = "add-tag"
+
+    logged_artifact = proj_obj.log_artifact(
+        "my-artifact",
+        body=b"some data",
+        tag=tag,
+    )
+
+    db.tag_artifacts(logged_artifact, project, tag_name=add_tag)
```

Comment:
All these lines are duplicated between the 3 tests, maybe move them to a helper function?